### DISCUSSION TITLE
Find the Hidden Number! :unlock:

### DISCUSSION THREAD

**Discussion by peymansf2000** [ParanoidUser/yolo#1272] [Galaxy Trivia] [answered]
Consider a positive integer n. If you multiply n by 3, add 7, and then divide the result by 2, you get the same number as if you had simply added 9 to n. What is the value of n?

**Comment by tomerf** [comment, marked as answer]
(3n + 7) / 2 = n + 9
3n + 7 = 2n + 18
n = 11

**Reply by peymansf2000** [reply]
Good job!🎉🎉
But I will wait for another person, that doesn't have the Galaxy Brain badge.

**Reply by tomerf** [reply]
lol, I'm aiming for the gold badge.

**Reply by peymansf2000** [reply]
OK, then ask some questions in the discussion too! I want that badge too!
:)

**Reply by peymansf2000** [reply]
If I mark my reply as the answer, does it contribute to earning the badge? 🤔 🤔

**Reply by tomerf** [reply]
If you answer your own question? I doubt that.
For the gold badge you need 32 accepted answers.
